<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeyondAbyssWidget</class>
 <widget class="QWidget" name="BeyondAbyssWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BeyondAbyssWidget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>1100</width>
     <height>570</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>570</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="allFileShowButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>所有文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recyleBinShowButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>回收站</string>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>900</width>
       <height>570</height>
      </rect>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>260</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>所有文件</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>260</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>回收站</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>570</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>正在上传</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>正在下载</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testUploadButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>上传</string>
      </property>
     </widget>
     <widget class="QPushButton" name="download">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>下载</string>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>900</width>
       <height>570</height>
      </rect>
     </property>
     <widget class="QWidget" name="page_6">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>280</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>上传</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_7">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>260</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>下载</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>300</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>分享页面</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>240</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>日志管理</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>250</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>权限管理</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="CustomStatusBar" name="status_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomStatusBar</class>
   <extends>QWidget</extends>
   <header location="global">customstatusbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
